src/pages/HomePage/MapSide/MapSide.jsx

```diff
@@ -42,14 +43,7 @@ function MapHouses() {
           <Typography className={styles.searchBy}>SEARCH BY</Typography>
         </ButtonSearch>
       </Box>
-      <Box
-        component="img"
-        sx={{
-          width: { xs: '90%', md: '90%', xl: '803px' },
-          height: { xl: '626px' },
-        }}
-        src="\src\assets\images\Map.png"
-      />
+      <GoogleMapComp style={{ height: '600px', width: '90%', minWidth: '420px' }} />
```

Comment:
It could be a good idea to assign a class instead - it could make managing styles easier.